README.md

```diff
@@ -52,15 +52,14 @@ A complete `.github/workflows/changelog.yml` might look like this:
 name: Check Changelog
 on:
   pull_request:
-    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]
-    branches:
-      - main
+    types: [assigned, labeled, opened, reopened, synchronize, unlabeled]
+    branches: [main]
 jobs:
   Check-Changelog:
     name: Check Changelog Action
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Generally I'd prefer not to push people to setups that might break, so I'd rather have a fixed version here.

Reply:
For such a simple action it should be fine to use ubuntu-latest. Usually the jobs are independent, I don't think anyone is going add more steps after the changelog action, so they'll use their preferred OS.